node-fs-readfile/read-dijkstra.js

```diff
@@ -0,0 +1,13 @@
+import { readFile } from 'node:fs/promises';
+
+async function readDijkstraFile() {
+  try {
+    const filePath = new URL('./dijkstra.txt', import.meta.url);
```

Comment:
This was in the sample code in the Node docs, but it isn't necessary. You could just pass the file name directly to `readFile`.